tests/unit/MDMRepetitionOverTimeTests.swift

```diff
@@ -0,0 +1,33 @@
+/*
+ Copyright 2017-present The Material Motion Authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import XCTest
+import MotionInterchange
+
+class MDMRepetitionOverTimeTests: XCTestCase {
+
+  func testInitializationWithDuration() {
+    let repetition = MDMRepetitionOverTime(duration: 5.5)
+    XCTAssertEqualWithAccuracy(repetition.duration, 5.5, accuracy: 0.001)
+    XCTAssertFalse(repetition.autoreverses)
+  }
+
+  func testInitializationWithNumberOfRepetitionsAndAutoreversed() {
```

Comment:
The name says "number of repetitions" but the code says duration. Wrong name?

Reply:
Fixed.